mavros/launch/px4_config.yaml

```diff
@@ -106,6 +106,12 @@ setpoint_position:
     rate_limit: 50.0
   mav_frame: LOCAL_NED
 
+# setpoint_raw
+setpoint_raw:
```

Comment:
Not raw, it is attitude.

Reply:
Yes, I forgot to move this when I changed it to the attitude.

Reply:
I suppose that whole entry for sp-raw not needed.